react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -56,10 +62,13 @@ class Thumbnail extends Component {
      * @returns {ReactElement}
      */
     render() {
-        const audioTrack = this.props._audioTrack;
-        const largeVideo = this.props._largeVideo;
-        const participant = this.props.participant;
-        const videoTrack = this.props._videoTrack;
+        const {
+            _audioTrack,
+            _hideVideo,
+            _largeVideo,
+            _videoTrack,
+            participant
+        } = this.props;
```

Comment:
can you destructure - rename instead, so the next changes are not necessary?

Reply:
Of course I could, but I think consistent, reasonable code is more important than to save a bunch of lines of changes. What do you think?